### DISCUSSION TITLE
When arm or x86 realese

### DISCUSSION THREAD

**Discussion by Slodziak190q** [gitpod-io/openvscode-server#182] [Q&A] [answered]
J Need it for raspberry pi

**Comment by filiptronicek** [comment]
Please try running the image specified here from a community member: https://github.com/gitpod-io/openvscode-server/issues/36#issuecomment-939210252 before we have it in our repository.

**Comment by filiptronicek** [comment, marked as answer]
You can now download releases for the Raspberry Pi, too! https://github.com/gitpod-io/openvscode-server/releases/tag/openvscode-server-v1.62.2

**Reply by Slodziak190q** [reply]
thank you!
